### PR TITLE
[Filebeat] Create Kibana dashboards for the ActiveMQ module

### PR DESCRIPTION
**Pull request by mtojek**
This PR introduces dashboards for the ActiveMQ filebeat module:
* audit events
* application events

Screenshots attached.

Master issue: https://github.com/elastic/beats/issues/14744

filebeat-activemq-application-events
filebeat-activemq-audit-events

### CODE REVIEW

**Comment by mtojek** [comment]
Changelog has been updated.

Honestly, I would keep them separate, because audit logs are disabled by default and if you don't want to use them, half of your screen would simply report nothing. I don't insist on this approach, so I'm also fine with combining them into a single piece.

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -436,6 +436,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add dashboards to the CEF module (ported from the Logstash ArcSight module).
 - Add dashboards to the CEF module (ported from the Logstash ArcSight module). {pull}14342[14342]
 - Fix timezone parsing in haproxy pipeline. {pull}14755[14755]
+- Add new module for ActiveMQ {pull}14880[14880]
```
Comment (ChrsMark) [inline CHANGELOG.next.asciidoc]:
```suggestion
- Add dashboards for ActiveMQ Filebeat modules{pull}14880[14880]
```

**Comment by ChrsMark** [comment]
> Changelog has been updated.
> 
> Honestly, I would keep them separate, because audit logs are disabled by default and if you don't want to use them, half of your screen would simply report nothing. I don't insist on this approach, so I'm also fine with combining them into a single piece.

Makes sense, I see the reason to keep it separately. Thanks for elaborating on this!

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (mtojek) [reply CHANGELOG.next.asciidoc]: Right, stupid mistake :D Thanks!

**Comment by mtojek** [comment]
Travis has just gone green. Merging.

**Pull request merged**